Remotion/ObjectBinding/Validation/BusinessObjectValidationResult.cs

```diff
@@ -52,9 +52,9 @@ private static IEnumerable<BusinessObjectValidationFailure> CreateBusinessObject
 
       switch (validationFailure)
       {
-        case PropertyValidationFailure propertyValidationFailure
+        case ValidationFailure propertyValidationFailure
             when validationFailure.ValidatedObject is IBusinessObject validatedBusinessObject
-                 && GetBusinessObjectProperty(validatedBusinessObject, propertyValidationFailure.ValidatedProperty) is { } validatedBusinessObjectProperty:
+                 && GetBusinessObjectProperty(validatedBusinessObject, propertyValidationFailure.ValidatedProperties.Single().Property) is { } validatedBusinessObjectProperty:
```

Comment:
We must never ever use `Single()` in production code. You can never analyse an error from this. 

The rule for "property validation failure" are validatedBusienssObject.Equals(validatedProperties.ValidatedObject) and validated.Properties.Count == 1. Please optimize the example

If this occurs multiple times, we may need (want) to add a property to ValidationFailure: `IsPropertyValidationFailure` that does that calculation. I do think I prefer to not have this, though, because it might expose more semantics than we like to for normal cases.